setup.py

```diff
@@ -0,0 +1,61 @@
+from setuptools import setup
+
+filename = 'rfhub2/version.py'
+exec(open(filename).read())
+
+setup(
+    name='rfhub2',
+    version=version,
+    author=['Pawel Bylicki', 'Maciej Wiczk'],
+    author_email=['pawelkbylicki@gmail.com', 'maciejwiczk@gmail.com'],
+    url='https://github.com/pbylicki/rfhub2/',
+    keywords='robotframework',
+    license='Apache License 2.0',
+    description='Webserver for robot framework and python assets documentation',
+    long_description=open('README.md').read(),
+    long_description_content_type='text/markdown',
+    zip_safe=True,
+    include_package_data=True,
+    install_requires=['aiofiles==0.4.0', 'Click==7.0', 'dataclasses==0.6', 'fastapi==0.30.0', 'h11==0.8.1',
```

Comment:
I would reduce it to only top-level dependencies, I think it is too strict and does not allow for Windows installation. It could be reduced to Click, fastapi, robotframework, sqlalchemy and requests. I would also relax the requirements for versions, and if required put lower-than/greater-than kind of constraints. It would require some testing, I think that there may be some breaking changes for fastapi dependencies since the development is heavy there. for robot probably version 3+ is enough. For the rest I think that any recent version should work.

Reply:
Sure thing. I will change it to more relaxed way. We can introduce dependabot or PyUp for managing dependencies. Probably after #3. What do You think?